mne/viz/_3d.py

```diff
@@ -2559,6 +2559,88 @@ def snapshot_brain_montage(fig, montage, hide_sensors=True):
     return proj.xy, im
 
 
+def plot_sensors_connectivity(raw, picks, ch_names, con):
+    """Visualize the sensor connectivity in 3D.
+
+    Parameters
+    ----------
+    raw : instance of Raw
+        A Raw object containing FIF data.
+    picks: array of int
+        Indices of selected channels.
```

Comment:
Move this after `con` and make the default `None` with `%(picks_data_good)s` or so

Reply:
When I tried to replace the docstring of `picks` by `%(picks_good_data)s` I got the following error:
```sh
E           AssertionError: 
E           mne.viz._3d.plot_sensors_connectivity %(picks_good_data)s != picks
```
How does it work?